app/controllers/bulk_upload_lettings_results_controller.rb

```diff
@@ -19,6 +19,10 @@ def resume
     end
   end
 
+  def summary
+    @bulk_upload = current_user.bulk_uploads.lettings.find(params[:id])
+  end
+
```

Comment:
Could we extract this to a `before_action :find_bulk_upload, only: %I[summary show resume]` etc?

Reply:
I'm not really a fan of this pattern. For example like https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/app/controllers/locations_controller.rb#L3 you end up with a load of intermingled `before_action`s and takes a while to work out state for a specific method. whereas here its super obvious and zero doubt